src/Service.Tests/DatabaseSchema-MsSql.sql

```diff
@@ -252,7 +253,17 @@ CREATE TABLE mappedbookmarks
 (
 	id int IDENTITY(1,1) PRIMARY KEY,
 	bkname nvarchar(50) NOT NULL
-) 
+)
+
+create table books_sold
+(
+  [id] int PRIMARY KEY not null,
+  [book_name] varchar(50),
+  [row_version] timestamp,
+  [copies_sold] int default 0,
+  [last_sold_on] datetime2(7),
```

Comment:
`last_sold_on` doesnt have a default value, so it will be nullified. So `last_sold_on_date` will also be null. 
 Which is the same effect if it were NOT a read only field. Can we set a default value to `last_sold_on` to say CURRENT_TIMESTAMP to test a non-null value gets assigned to `last_sold_on_date`. 

 At the same time, we already have a table `stocks_price` which has `instant` column with a default value of CURRENT_TIMESTAMP. This means we can add a column to that table to test the same functionality. Trying to reduce the number of additional tables we need for testing.

Reply:
@Aniruddh25 Even though that table can ideally be used, because of this bug: https://github.com/Azure/data-api-builder/pull/1636, we cannot use that right now unless we wait for that PR to be merged first.

Reply:
Proceeding with this table only since the PR: https://github.com/Azure/data-api-builder/pull/1636 is still left with some decision making first.